components/flipper.py

```diff
@@ -2,18 +2,25 @@
 
 
 class Flipper:
+    """this flips a bucket with an arm that has a spinning motor"""
+    
 
     flipper_motor = wpilib.Talon
     spinning = False
+    #It starts not spinning
 
     def turn_on(self):
         self.spinning = True
 
+        
```

Comment:
We don't need the extra line.

Tip: `autopep8` or `yapf` will automatically format your code

Reply:
You still have an extra line here